lib/cdo/honeybadger.rb

```diff
@@ -84,4 +84,30 @@ def self.parse_exception_dump(error)
 
     [error_message, error_lines]
   end
+
+  # Returns all issues (across cronjobs, dashboard, and pegasus) that have occured since midnight.
+  # @return [Array[Hash]] An array of hashes summarizing the recent issues.
+  def self.get_recent_issues
+    issues = []
+
+    midnight_epoch = DateTime.now.to_i / 86400 * 86400
+    {cronjobs: 45435, dashboard: 3240, pegasus: 34365}.each do |project, project_id|
+      response = `curl -u #{CDO.honeybadger_api_token}: https://app.honeybadger.io/v2/projects/#{project_id}/faults?occurred_after=#{midnight_epoch}`
+      parsed_response = JSON.parse response
+      parsed_response['results'].each do |issue|
+        next if issue['resolved']
+        next if issue['ignored']
```

Comment:
Note the API response seems to not have information about whether the issue is paused. Going to confirm this, making a feature request if this is indeed the case.